src/winforms/toga_winforms/widgets/selection.py

```diff
@@ -0,0 +1,40 @@
+from toga_winforms.libs import WinForms
+
+from .base import Widget
+
+
+class TogaComboBox(WinForms.ComboBox):
+    def __init__(self, interface):
+        super().__init__()
+        self.interface = interface
+        self.SelectedIndexChanged += self.on_select
+
+    def on_select(self, sender, event):
+        if self.interface.on_select:
+            self.interface.on_select(self.interface)
+
+
+class Selection(Widget):
+    def create(self):
+        self.native = TogaComboBox(self.interface)
```

Comment:
Is Combobox the right widget here? My understanding was that ComboBox allowed you to enter text that wasn't specifically on the list for selection - it's a "combination" of a selection and a text box. Does this mean users could set a value that isn't on the list of items that has been provided?

Reply:
Thank you for clear explanation, as always. 
I changed a property of ComboBox so that it's not editable.